packages/lit-dev-content/package.json

```diff
@@ -9,124 +9,88 @@
   },
   "wireit": {
     "build": {
-      "dependencies": [
-        "build:eleventy"
-      ]
-    },
-    "dev:build": {
-      "dependencies": [
-        "dev:build:eleventy"
-      ]
-    },
-    "prod:build": {
+      "command": "LITDEV_ENV=local NODE_OPTIONS=--experimental-vm-modules eleventy",
       "dependencies": [
         "prod:build:assets"
       ],
       "files": [
         "site/**",
         ".eleventy.js"
       ],
-      "command": "NODE_OPTIONS=--experimental-vm-modules eleventy"
-    },
-    "dev:serve": {
-      "dependencies": [
-        "../lit-dev-tools-esm:build",
-        "dev:build"
-      ],
-      "command": "node ../lit-dev-tools-esm/lib/dev-server.js --open"
-    },
-    "build:eleventy": {
-      "dependencies": [
-        "prod:build:assets"
-      ],
       "output": [
         "_site"
-      ],
-      "files": [
-        "site/**",
-        ".eleventy.js"
-      ],
-      "command": "LITDEV_ENV=local NODE_OPTIONS=--experimental-vm-modules eleventy"
+      ]
     },
-    "build:eleventy:watch": {
+    "dev:build": {
+      "command": "LITDEV_ENV=dev NODE_OPTIONS=--experimental-vm-modules eleventy --quiet",
       "dependencies": [
-        "prod:build:assets"
-      ],
-      "output": [
-        "_site"
+        "build:ts",
+        "fonts:manrope",
+        {
+          "script": "build:samples",
+          "cascade": false
+        }
       ],
-      "#comment": "This is awkward. We want to do the watch part for build:assets ourselves, but let eleventy do its own watch mode itself. Maybe --incremental will do a quick rebuild of just the changed parts, and we don't need --watch?",
       "files": [
         "site/**",
         ".eleventy.js"
       ],
-      "command": "LITDEV_ENV=local NODE_OPTIONS=--experimental-vm-modules eleventy --watch --incremental"
-    },
-    "dev:build:eleventy": {
-      "dependencies": [
-        "dev:build:assets"
-      ],
       "output": [
         "_dev"
-      ],
-      "files": [
-        "site/**",
-        ".eleventy.js"
-      ],
-      "command": "LITDEV_ENV=dev NODE_OPTIONS=--experimental-vm-modules eleventy"
+      ]
     },
-    "dev:build:eleventy:watch": {
+    "prod:build": {
+      "command": "NODE_OPTIONS=--experimental-vm-modules eleventy",
       "dependencies": [
-        "dev:build:assets"
-      ],
-      "output": [
-        "_dev"
+        "prod:build:assets"
       ],
       "files": [
         "site/**",
         ".eleventy.js"
-      ],
-      "command": "LITDEV_ENV=dev NODE_OPTIONS=--experimental-vm-modules eleventy --watch --incremental"
+      ]
     },
-    "dev:build:assets": {
+    "dev:serve": {
+      "command": "node ../lit-dev-tools-esm/lib/dev-server.js --open",
+      "service": true,
       "dependencies": [
-        "build:ts",
-        "build:samples",
-        "fonts:manrope"
+        "../lit-dev-tools-esm:build",
+        "../../:start:fake-github",
+        {
+          "script": "dev:build",
+          "cascade": false
```

Comment:
Wonder if we'll ever have need for cascade to only cascade on a single dep or a subset of deps.

Reply:
Yeah like if we did SSR on the server live (as we are doing for webcomponents.org), then the JS for the server renderable components would need to restart the server potentially. In this case we don't need to.

Reply:
What i'm thinking in that case would be it'd be cool if cascade could be of type `boolean|string[]` where the array would be `"cascade": ["build:ssr"]`

Reply:
It already works like that, but with a different syntax. In this case, "../../:start:fake-github" has default cascade:true, but "dev:build" has cascade:false.

Reply:
(Which made me realize start:fake-github can also be cascade, since it doesn't require a restart either)

Reply:
So i'm imagining something like

```json
"server": {
  "command": "node server.js",
  "dependencies": [
    "build:ssr",
    "build:ts"
  ],
  cascade: [
    "build:ssr"
  ]
}
```

so if ts rebuilds the client-side TS it shouldn't _have_ to reload. Or unless I'm misunderstanding how cascade works across rules

Reply:
also maybe duplicating rules across two fields like cascade and deps is bad...

Reply:
By default a dependency triggers reload (`cascade:true`). If you want a dependency to not trigger reload, you set `cascade:false` (on that specific dependency). Does that help? I think your example is an alternative syntax to what's already supported? Or maybe I am misunderstanding.

Reply:
oh! okay I did misunderstand. So any services above it will not trigger?

e.g.

```
┌───────────┐         ┌─────────┐
│ Service 1 ├────────►│Service 2│
└───────────┘         └─┬───────┘
                        │
                        │
                        ├─────────────┐
                        │             │
                        │             │
                        │             │
         ┌──────────────▼──┐   ┌──────▼────────┐
         │       Rule 1    │   │      Rule 2   │
         │                 │   │               │
         │  cascade: false │   │ cascade: true │
         └─────────────────┘   └───────────────┘
```

if rule 1 changes then service 1 and 2 do not change, but if rule 2 changes they both do?

Reply:
Yep that's right!

And if you wanted for some reason e.g. rule 1 changes to cause service 1 to restart, but not service 2, then you'd also add a dependency between service 1 and rule 1, with cascade:false.

Reply:
(Edited above comment, had a typo)

Reply:
uhh like this?

```
              ┌───────────┐         ┌─────────┐
              │ Service 1 ├────────►│Service 2│
              └────┬──────┘         └─┬───────┘
                   │                  │
                   │                  │
                   │                  ├─────────────┐
       ┌───────────┘                  │             │
       │                              │             │
       │                              │             │
┌──────▼────────┐      ┌──────────────▼──┐   ┌──────▼────────┐
│    Rule 1.5   │      │     Rule 1      │   │    Rule 2     │
│               ├─────►│                 │   │               │
│ cascade: true │      │  cascade: false │   │ cascade: true │
└───────────────┘      └─────────────────┘   └───────────────┘
```

Reply:
Kinda, but `cascade` is as annotation on the edge not the script, so it's simpler to think of it like this:

```
   ┌───────────┐         ┌─────────┐
   │ Service 1 ├────────►│Service 2│
   └────┬──────┘         └─┬────┬──┘
        │                  │    │
        │                  │    └────────┐
        │                  │             │
        │           [cascade:false]      │
        │                  │             │
        │   ┌──────────────▼──┐   ┌──────▼────────┐
        └──►│     Rule 1      │   │    Rule 2     │
            └─────────────────┘   └───────────────┘
```